### DISCUSSION TITLE
Option to change description font size

### DISCUSSION THREAD

**Discussion by rom428** [TechbeeAT/jtxBoard#1163] [Ideas]
In my opinion, the description font size is too small, especially if compared with other journal apps. I'd like to have an option on the settings page to make it bigger.

**Comment by patrickunterwegs** [comment]
Hi @rom428 , sorry for the late reply, I lost a bit track of the discussions. 
I have added an accessibility mode option lately, this would increase the font size for very small elements. Does that help?
Otherwise I have now also added an option to change the font, e.g. to the Noto-font. Could you try it? It generally appears a bit larger than the default Roboto-font.

**Reply by rom428** [reply]
Hi, I've been using the noto font and it does help. Regarding the accessibility mode, I see no effect at all, maybe it's a bug. Could you send me a screenshot of the app with the accessibility mode on?

**Reply by patrickunterwegs** [reply]
Hi @rom428 , the effect of the accessibility mode is mainly on the badges (e.g. in the list view) as they use a very small font size by default. If you use another view, you might not see a difference.
